Dockerfile

```diff
@@ -18,9 +18,7 @@
 RUN pip install --upgrade pip && \
     pip install -r requirements.txt
 
-RUN python manage.py migrate
-
 EXPOSE 8080
 
 # Run database migrations and start the Django application
-ENTRYPOINT ["python", "manage.py", "runserver", "0.0.0.0:8080"]
+ENTRYPOINT ["sh", "-c", "python manage.py migrate && python manage.py runserver 0.0.0.0:8080"]
```

Comment:
Using `sh -c` in the ENTRYPOINT can make it harder to pass arguments to the container. Consider using a shell script or directly using `python manage.py` commands without `sh -c` for better flexibility.